### ISSUE TITLE
Timestamp format configurable

### ISSUE THREAD

**Issue by benurb** [detro/consoleplusplus#1]
First of all. Your lib looks really cool and makes the console look a lot stylisher :+1: 
Though there is one thing I would really like to see: Configurable timestamps, meaning that you give him a standard date format string (don't know how it's called exactly, but it's mostly the same accross programming languages: e.g. mm-dd-yyyy) and it uses that as output. The current timestamp is a bit long imo and contains useless information for me. The time is for example enough for me, but there might be other applications that need another format.

To sum it up: It would be really cool to have configurable timestamp formats :-)
